Makefile

```diff
@@ -20,9 +20,11 @@ install-build-deps: ## Install dependencies (packages and tools)
 
 ##@ Build
 
-.PHONY: build
-build: generate-bindata-assets generate-kubernetes-types  ## Build eksctl
-	CGO_ENABLED=0 go build -tags "$(GO_BUILD_TAGS)" -ldflags "-X $(version_pkg).gitCommit=$(git_commit) -X $(version_pkg).builtAt=$(built_at)" ./cmd/eksctl
+godeps_cmd = go list -deps -f '{{if not .Standard}}{{ $$dep := . }}{{range .GoFiles}}{{$$dep.Dir}}/{{.}} {{end}}{{end}}' $(1) | sed "s%${PWD}/%%g"
+godeps = $(shell $(call godeps_cmd,$(1)))
```

Comment:
Should this be an immediate assignment, or it's intended to be evaluated each time?

Reply:
Never mind, I see the usage now.

Reply:
Yeah, you cannot evaluate it immediately since it has a parameter.